### ISSUE TITLE
npm package name

### ISSUE THREAD

**Issue by asednev** [mogsie/xstate-plantuml#1]
Has this been published to npm? If so, what's the package name.

Thanks

**Comment by mogsie** [comment]
No, I haven't taken that plunge quiet yet! :D  But you're the second to suggest this, so I guess I'll have to.
